src/app/[lang]/my-bookings/page.tsx

```diff
@@ -0,0 +1,102 @@
+'use client';
+import clsx from 'clsx';
+import { useState } from 'react';
+// components
+import { Tabs, Typography, Avatar, Icon } from '@/modules/core/components';
+
+import styles from './my-bookings.module.scss';
+
+export default function MyBookings() {
+  const [value, setValue] = useState('bookings');
+
+  return (
+    <div className={styles.root}>
+      <section className={clsx(styles.section)}>
+        <div className={styles.title}>
+          <Typography variant='title'>My bookings</Typography>
+        </div>
+
+        <Tabs
+          value={value}
+          onChange={setValue}
+          tabs={[
+            {
+              key: 'bookings',
+              name: 'Bookings',
+              icon: 'list',
+            },
+            {
+              key: 'calendar',
+              name: 'Calendar',
+              icon: 'calendar',
+            },
+          ]}
+        />
+
+        {value === 'bookings' && (
+          <div className={styles.tabContiner}>
+            <div className={styles.tabBox}>
+              <Typography variant='subtitle'>Upcoming bookings</Typography>
+              <div className={styles.bookingContainer}>
+                <div className={styles.bookingInfo}>
```

Comment:
Create separate component for booking card. Create module 'booking'. Create component there - `BookingInfoCard`. Put all necessary data into props (e.g. date, name, service name ... ). Also, use date-fns for formatting `Date` into "Thursday, Jun 01, 12:00 - 13:00". To achieve this, take a look at `format` method - https://date-fns.org/v2.30.0/docs/format